examples/demo-app/src/actions.js

```diff
@@ -305,3 +329,51 @@ export function loadSampleConfigurations(sampleMapId = null) {
   }
 }
 
+/**
+ * this action will be triggered when the file is being uploaded
+ * @param isLoading
+ * @param metadata
+ * @returns {{type: string, isLoading: *, metadata: *}}
+ */
+export function setPushingFile(isLoading, metadata) {
+  return {
+    type: PUSHING_FILE,
+    isLoading,
+    metadata
+  };
+}
+
+/**
+ * This method will export the current kepler config file to the choosen cloud platform
+ * @param data
+ * @param handlerName
+ * @returns {Function}
+ */
+export function exportFileToCloud(data, handlerName = 'dropbox') {
+  const authHandler = AUTH_HANDLERS[handlerName];
+  return dispatch => {
+    // we are exporting to json format with 2 spaces,
+    // we could save bandwidth if we used a single line
+    // but it wouldn't be readable
+    const newBlob = new Blob([JSON.stringify(data, null, 2)], {type: 'application/json'});
```

Comment:
if you don't add `null, 2` it would be a smaller string, smaller fine

Reply:
done